Exiled.API/Extensions/RoleExtensions.cs

```diff
@@ -60,6 +60,7 @@ public static class RoleExtensions
         /// </summary>
         /// <param name="roleType">The <see cref="RoleTypeId"/>.</param>
         /// <returns><see cref="Team"/>.</returns>
+        [Obsolete("Use PlayerRolesUtils.GetTeam(roleType) in NW-API", false)]
```

Comment:
```suggestion
        [Obsolete("Use PlayerRolesUtils.GetTeam(roleType) instead", false)]
```
my bad, it's in the assembly